packages/office-ui-fabric-react/src/components/Persona/Persona.scss

```diff
@@ -317,8 +334,58 @@ $ms-Persona-presenceSizeXl: 28px;
 
 //== Modifier: Extra Small Persona
 //
+.ms-Persona.ms-Persona--xxs {
+  height: $ms-Persona-sizeXxs;
+  min-width: $ms-Persona-sizeXxs;
+  margin-right: 4px;
+
+  .ms-Persona-imageArea,
+  .ms-Persona-image {
+    flex: 0 0 $ms-Persona-sizeXxs;
+    height: $ms-Persona-sizeXxs;
+    width: $ms-Persona-sizeXxs;
+  }
+
+  .ms-Persona-placeholder {
+    font-size: 18px;
+    top: 4px;
+  }
+
+  .ms-Persona-initials {
+    font-size: $ms-font-size-xs;
+    height: $ms-Persona-sizeXxs;
+    line-height: $ms-Persona-sizeXxs;
+  }
+
+  .ms-Persona-presence {
+    height: $ms-Persona-presenceSizeXxs;
+    width: $ms-Persona-presenceSizeXxs;
+    border: 1px solid #ffffff;
+    @include left(16px);
+  }
+
+  .ms-Persona-presenceIcon {
+    font-size: 6px;
+    line-height: 9px;
+  }
+
+  .ms-Persona-primaryText {
+    font-size: $ms-font-size-m;
+    padding-top: 3px;
+  }
+
+  .ms-Persona-secondaryText {
+    display: none;
+  }
+}
+
+
```

Comment:
Yeah I got mixed up, sorry about that. But the persona tests serves as a helpful example I hope for writing facepile tests